sum.js

```diff
@@ -0,0 +1,9 @@
+function sum(args){
+    let sum_of_args = 0;
+    for (const i of args){
+        sum_of_args = sum_of_args + i
+    }
+    return sum_of_args;
+}
+
+console.log(sum([2, 3]))
```

Comment:
```suggestion
const sum = (...args) => {
  let total = 0;
  for (const x of args) {
    total += x;
  }
  return total;
};

const ages = [2, 3];
const result = sum(ages);
console.log({ result });
```

Reply:
- Use `camelCase` in js
- Avoid complex expressions, use intermediate variables
- Add `;` after statements